keyboards/vortex/m0110a/rules.mk

```diff
@@ -0,0 +1 @@
+WEAR_LEVELING_DRIVER = spi_flash
```

Comment:
```suggestion
# This file intentionally blank
```

Reply:
@waffle87 By making these changes, compilation is terminating...

Reply:
@gskygithub The code that enables this change doesn't exist on the target branch yet. This suggestion can be ignored.